docs/en/contributing-to-appium/developers-overview.md

```diff
@@ -0,0 +1,149 @@
+## A Developer's Overview of Appium
+
+(Before reading this document, please ensure that you have read and understood
+the more general [introduction to Appium
+concepts](/docs/en/about-appium/intro.md) and the more general [contribution
+instructions](/CONTRIBUTING.md).)
+
+### Technical Vision
+
+Appium aims to be a Mobile JSONWP front-end for the best app automation
+technologies. That's it. We want to take all the different useful automation
+engines and smooth away their differences and gotchas by making Appium drivers
+for them and bringing them under the umbrella of Appium itself. This is very
+similar to the goal of the Selenium project. For our part, we want to have
+every driver be an independent entity (separate repo, tests, etc...) even as it
+uses shared libraries that make the development of an Appium driver as simple
+and boilerplate-free as possible. We use modern JavaScript because JavaScript
+is everywhere and it's easy for many developers to understand and contribute
+back to.
+
+### Developer Community
+
+Anyone is welcome to become an Appium developer; just read this guide and get
+some of your code merged, and you are one of us! If you stick around and help
+a lot, we will also make you a committer so that you can continue to help the
+community more easily. If you are developing code for Appium and have
+questions, reach out to the developer community at
+`appium-developers@googlegroups.com`. Note that this is a mailing list for
+_development_ questions, not _usage_ questions or bug reports. For those, go to
```

Comment:
> _development_ questions, not _usage_ questions or bug reports. For those, go to
> our forums or GitHub issue tracker respectively.

Since this is an ongoing issue, It'd be better to clearly state what goes where. Something like this:

> _development_ questions, not _usage_ questions or bug reports. Usage questions belong on our forums. The GitHub issue tracker is exclusively used for bug reports and feature requests.